router/proxy/transport.go

```diff
@@ -88,7 +118,7 @@ func (t *transport) RoundTrip(req *http.Request) (*http.Response, error) {
 	return nil, errNoBackends
 }
 
-func (t *transport) Connect(remoteAddr net.Addr) (net.Conn, error) {
+func (t *transport) Connect() (net.Conn, error) {
```

Comment:
I assumed that this argument was there to support the PROXY protocol in the future, but I think removing it for now is fine.